addons/binding/org.openhab.binding.toon/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="toon"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+                  
+
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Current room temperature</description>
+        <category>temperature</category>
+        <state readOnly="true" pattern="%.1f °C">
+        </state>
+    </channel-type>
+    
+    <channel-type id="setpoint">
+        <item-type>Number</item-type>
+        <label>Setpoint</label>
+        <description>Thermostat setpoint</description>
+        <category>heating</category>
+        <state min="4.5" max="30" step="0.1" pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+    <channel-type id="setpointMode">
+        <item-type>Number</item-type>
+        <label>Setpoint Mode</label>
+        <description>Chosen setpoint mode (comfort, active, sleep, away)</description>
+        <category>heating</category>
+        <state readOnly="false">
+            <options>
+                <option value="-1">override`</option>
```

Comment:
The labels might look better in the UI is they start with a capital, what do you think?